dmscripts/notify_suppliers_of_new_questions_answers.py

```diff
@@ -0,0 +1,159 @@
+import dmapiclient
+
+from datetime import datetime, date, timedelta
+
+from dmscripts.helpers import env_helpers, logging_helpers
+from dmscripts.helpers.html_helpers import render_html
+from dmscripts.helpers.logging_helpers import logging
+from dmutils.email.dm_mandrill import send_email
+from dmutils.formats import DATETIME_FORMAT
+
+
+EMAIL_SUBJECT = "New question and answer responses on the Digital Marketplace"
```

Comment:
The comma is probably not worth mentioning, but I think the subject line is, it definitely seems odd to me.